commit/operator/operator.pyx

```diff
@@ -26,6 +26,45 @@ cdef extern void COMMIT_At(
     unsigned char *_ICthreadsT, unsigned int *_ECthreadsT, unsigned int *_ISOthreadsT
 ) nogil
 
+cdef extern void COMMIT_L1(
```

Comment:
Can we call these functions something like `Tikhonov_L1`, `Tikhonov_L2` etc? This way they clearly convey the meaning of what they are